src/Smalot/PdfParser/Page.php

```diff
@@ -730,6 +749,18 @@ public function getDataTm(?array $dataCommands = null): array
                     $Ty = 0;
                     break;
 
+                case 'cm':
+                    $newConcatTm = (array) explode(' ', $command['c']);
+                    $TempMatrix = [];
+                    // Multiply with previous concatTm
+                    $TempMatrix[0] = (float) $concatTm[0] * (float) $newConcatTm[0] + (float) $concatTm[1] * (float) $newConcatTm[2];
+                    $TempMatrix[1] = (float) $concatTm[0] * (float) $newConcatTm[1] + (float) $concatTm[1] * (float) $newConcatTm[3];
+                    $TempMatrix[2] = (float) $concatTm[2] * (float) $newConcatTm[0] + (float) $concatTm[3] * (float) $newConcatTm[2];
+                    $TempMatrix[3] = (float) $concatTm[2] * (float) $newConcatTm[1] + (float) $concatTm[3] * (float) $newConcatTm[3];
+                    $TempMatrix[4] = (float) $concatTm[4] * (float) $newConcatTm[0] + (float) $concatTm[5] * (float) $newConcatTm[2] + (float) $newConcatTm[4];
+                    $TempMatrix[5] = (float) $concatTm[4] * (float) $newConcatTm[1] + (float) $concatTm[5] * (float) $newConcatTm[3] + (float) $newConcatTm[5];
```

Comment:
Just looked into the code once more. Can you give me a short summary of the indexes used in the multiplication here?

Reply:
They are the letters a to f as defined in the PDF Specficiation:
[0] = a
[1] = b
[2] = c
[3] = d
[4] = e
[5] = f
![image](https://github.com/user-attachments/assets/86366052-a6bd-435e-a603-94331e9acf42)
Source: https://opensource.adobe.com/dc-acrobat-sdk-docs/pdfstandards/PDF32000_2008.pdf 
Page: 128 (written page in corner of pdf: 120)